test/monarch/camp_monarch_interface.F90

```diff
@@ -20,194 +19,124 @@ module camp_monarch_interface
-    !> Initial concentration data
-    type(property_t), pointer :: init_conc_data => null( )
-    !> Interface input data
-    type(property_t), pointer :: property_set => null( )
-    !> Solve multiple grid cells at once?
+    type(property_t), pointer :: init_conc_data
+    type(property_t), pointer :: property_set
+    type(rxn_update_data_photolysis_t), allocatable :: photo_rxns(:)
+    real(kind=dp), allocatable :: base_rates(:),specs_emi(:),offset_photo_rates_cells(:)
+    integer :: n_photo_rxn
+    integer :: nrates_cells
     logical :: solve_multiple_cells = .false.
+    type(string_t), allocatable :: kpp_rxn_labels(:)
+    real(kind=dp) :: KPP_RSTATE(20)
+    integer :: KPP_ICNTRL(20)
   contains
-    !> Integrate PartMC for the current MONARCH state over a specified time step
     procedure :: integrate
-    !> Get initial concentrations (for testing only)
     procedure :: get_init_conc
-    !> Get monarch species names and ids (for testing only)
-    procedure :: get_MONARCH_species
-    !> Print the PartMC-camp data
-    procedure :: print => do_print
-    !> Load interface data from a set of input files
     procedure, private :: load
-    !> Create the PartMC <-> MONARCH species map
     procedure, private :: create_map
-    !> Load the initial concentrations
     procedure, private :: load_init_conc
-    !> Finalize the interface
     final :: finalize
-  end type monarch_interface_t
+  end type camp_monarch_interface_t
 
-  !> PartMC <-> MONARCH interface constructor
-  interface monarch_interface_t
+  interface camp_monarch_interface_t
     procedure :: constructor
-  end interface monarch_interface_t
+  end interface camp_monarch_interface_t
 
-  !> MPI node id from MONARCH
-  integer(kind=i_kind) :: MONARCH_PROCESS ! TODO replace with MONARCH param
-  ! TEMPORARY
+  integer(kind=i_kind) :: MONARCH_PROCESS
   real(kind=dp), public, save :: comp_time = 0.0d0
-
+  real(kind=dp), parameter :: mwair = 28.9628 !mean molecular weight for dry air [ g/mol ]
+  real(kind=dp), parameter :: mwwat = 18.0153 ! mean molecular weight for water vapor [ g/mol ]
 contains
 
 
-!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-
-  !> Create and initialize a new monarch_interface_t object
-  !!
-  !! Create a monarch_interface_t object at the beginning of the  model run
-  !! for each node. The master node should pass a string containing the path
-  !! to the PartMC confirguration file list, the path to the interface
-  !! configuration file and the starting and ending indices for chemical
-  !! species in the tracer array.
-  function constructor(camp_config_file, interface_config_file, &
-                       starting_id, ending_id, n_cells, mpi_comm) result (new_obj)
-
-    !> A new MONARCH interface
-    type(monarch_interface_t), pointer :: new_obj
-    !> Path to the PartMC-camp configuration file list
+  function constructor(camp_config_file, output_file_title, &
```

Comment:
why were the comments removed from this file?

Reply:
It was easier for me to understand the file. I wanted to just put the minimum and work from there.